src/Storages/MergeTree/IMergeTreeDataPart.cpp

```diff
@@ -193,6 +229,15 @@ void IMergeTreeDataPart::MinMaxIndex::merge(const MinMaxIndex & other)
     }
 }
 
+void IMergeTreeDataPart::MinMaxIndex::replace(const MinMaxIndex & other)
+{
+    if (!other.initialized)
```

Comment:
But is it possible to into `replace` for non initialized `other` ?

Reply:
And why replace, if this is just a copy?

Reply:
If an instance of un-initialized `IMergeTreeDataPart::MinMaxIndex` is passed, then `other` would be non initialized, right?

Well.. it's writing into the destination, not creating a copy and returning it to the caller. That's the way I see it, but happy to change the name if necessary.

Reply:
>If an instance of un-initialized IMergeTreeDataPart::MinMaxIndex is passed, then other would be non initialized, right?

Yes, but is this situation is ever possible?

Reply:
Possible? yes, but there isn't such case as of now. In any case, I replaced that with an assignment operator